src/Http/Client/Adapter/Curl.php

```diff
@@ -108,14 +108,14 @@ public function buildOptions(RequestInterface $request, array $options): array
                 $out[CURLOPT_CUSTOMREQUEST] = $request->getMethod();
                 break;
         }
-        
```

Comment:
Thanks @stickler-ci that was fast. I was wondering where these went, I was gonna do the same! :D